homework-02/task-03.js

```diff
@@ -0,0 +1,18 @@
+const findLongestWord = function (string) {
+  let longestWord = '';
+  for (let i = 0; i < string.split(' ').length; i++) {
+    if (string.split(' ')[i].length > longestWord.length) {
```

Comment:
Це потрібно винести в змінну пере циклом 
`const words = string.split(' ');`
а потім проходити циклом по масиву із слів

Reply:
В такому випадку це буде виглядати ось так:
```suggestion
    let longestWord = '';
    const words = string.split(' ');
    for(let i=0; i < words.length; i+=1) {
       if (words[i].length > longestWord.length) {
           longestWord = words[i];
       }
    }
```